src/Stripe.net/Services/Payouts/StripePayoutService.cs

```diff
@@ -9,6 +9,10 @@ public class StripePayoutService : StripeBasicService<StripePayout>
     {
```

Comment:
I like the way we use expandables right now, I don't think that needs to be changed. The only issue is the expandable property with a JsonConverter (e.g: SourceConverter) because that would never be called, since the deserialization happens on the internal property and the object is set in the Map method of the ExpandableProperty class.

For instance, with StripeCustomer (below), the property DefaultSource was always null because we deserialize the Source like json into the InternalDefaultSource which receives the value of "default_source" from the payload, then we handle other 2 properties (DefaultSourceId and DefaultSource) inside the mapper ExpandableProperty<Source>.Map by setting them individually after deserializing the object into T. If we keep the JsonConverter attribute in the property, that won't be called when settings the properties in the mapper. The way to fix that is by moving the JsonConverter to the class level of Source (my last commit: 907f36f8c3dae0f414e9bca909611adb5845713c)

```
public string DefaultSourceId { get; set; }

[JsonIgnore]
[JsonConverter(typeof(SourceConverter))]
public Source DefaultSource { get; set; }

[JsonProperty("default_source")]
internal object InternalDefaultSource
{
	set
	{
		ExpandableProperty<Source>.Map(value, s => DefaultSourceId = s, o => DefaultSource = o);
	}
}
```